src/assets/scripts/airport/AirportModel.js

```diff
@@ -102,43 +101,65 @@ export default class AirportModel {
         this.parse(options);
     }
 
+    /**
+     * @property real_fixes
+     * @return {array<FixModel>}
+     */
+    get real_fixes() {
+        return FixCollection.findRealFixes();
+    }
+
+    /**
+     * @property elevation
+     * @return {number}
+     */
     get elevation() {
         return this.position.elevation;
     }
 
+    /**
+     * @property magnetic_north
+     * @return {number}
+     */
     get magnetic_north() {
-        return degreesToRadians(this.position.magnetic_north);
+        return this.position.magneticNorthInRadians;
     }
 
+    /**
+     * @for AirportModel
+     * @method parse
+     * @param data {object}
+     */
     parse(data) {
         this.setCurrentPosition(data.position, data.magnetic_north);
 
-        this.name = _get(data, 'name', null);
-        this.icao = _get(data, 'icao', null);
-        this.radio = _get(data, 'radio', null);
+        this.name = _get(data, 'name', this.name);
+        this.icao = _get(data, 'icao', this.icao);
+        this.radio = _get(data, 'radio', this.radio);
+        this.level = _get(data, 'level', this.level);
         this.has_terrain = _get(data, 'has_terrain', false);
+        this.sids = _get(data, 'sids', {});
         this.stars = _get(data, 'stars', {});
```

Comment:
Would `airport.sids` and `airport.stars` not be deprecated by `airport.sidCollection` and `airport.starCollection`? Off the top of my head, I don't believe the *data* itself parsed from the airport files is really of any use once it's built into the `StandardRouteCollection`, from which the various methods are called. I'm not sure, but just figured I'd bring it up since I see both members are present.